views/about.erb

```diff
@@ -0,0 +1,14 @@
+<html>
+  <head>
+  <title>About Kelly</title>
+  </head>
+  <body>
+    <a href="/">Home</a>
+      <h1>About</h1>
+      <img src = "self.jpg" width="625"/>
```

Comment:
In ruby we add spaces before/after our `=` but in HTML we don't want to add those additional spaces